src/mapml-viewer.js

```diff
@@ -656,6 +711,20 @@ export class MapViewer extends HTMLElement {
       }
     });
   }
+  locate(options){
```

Comment:
Seems like the locate method does not work, if the user has interacted with the geolocation control in anyway:


https://user-images.githubusercontent.com/55751566/228655623-61120617-cc6d-470d-a220-d68c1ed5dd03.mp4

Reply:
What is the `options` parameter? It would be good to add some comments in the code showing/linking to what the options parameter might be.